<details>
<summary>Pre-Commit Test failed! Click here</summary>

```
[INFO] Initializing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Initializing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for git://github.com/pycqa/pydocstyle.git.
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://github.com/pre-commit/mirrors-mypy.
[INFO] Initializing environment for https://github.com/psf/black.
[INFO] Initializing environment for https://gitlab.com/PyCQA/flake8.
[INFO] Initializing environment for https://gitlab.com/PyCQA/flake8:pep8-naming.
[INFO] Installing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for git://github.com/pycqa/pydocstyle.git.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pre-commit/mirrors-mypy.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/psf/black.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://gitlab.com/PyCQA/flake8.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Tabs remover.............................................................Passed
Trim Trailing Whitespace.................................................Passed
Check for merge conflicts................................................Passed
Fix End of Files.........................................................Passed
Tests should end in _test.py.............................................Passed
Check for added large files..............................................Passed
Check for byte-order marker..............................................Passed
Check for case conflicts.................................................Passed
Check docstring is first.................................................Passed
Check JSON...............................................................Passed
Check for broken symlinks............................(no files to check)Skipped
Detect Private Key.......................................................Passed
Check python ast.........................................................Passed
Debug Statements (Python)................................................Passed
pydocstyle...............................................................Passed
Check Toml...............................................................Passed
Check Yaml...............................................................Passed
Fix End of Files.........................................................Passed
Trim Trailing Whitespace.................................................Passed
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

thoth/report_processing/components/solver.py:156:60: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters  [type-arg]
thoth/report_processing/components/solver.py:166:63: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters  [type-arg]
thoth/report_processing/components/solver.py:187:59: error: Implicit generic "Any". Use "typing.Dict" and specify generic parameters  [type-arg]
thoth/report_processing/components/solver.py:187:68: error: Implicit generic "Any". Use "typing.List" and specify generic parameters  [type-arg]
thoth/report_processing/components/solver.py:203:68: error: Implicit generic "Any". Use "typing.List" and specify generic parameters  [type-arg]
Found 5 errors in 1 file (checked 8 source files)

black....................................................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

thoth/report_processing/components/solver.py:29:1: F401 'thoth.storages.CephStore' imported but unused
```
</details>

--- a/tests/adviser_test.py
+++ b/tests/adviser_test.py
@@ -63,9 +63,7 @@ class TestAdviser(ReportProcessingTestCase):
             adviser_version=adviser_version, adviser_files=adviser_files
         )
 
-        final_dataframe, info_dataframe, error_dataframe = Adviser.create_summary_dataframes(
-            adviser_dataframe=adviser_dataframe
-        )
+        final_dataframe = Adviser.create_summary_dataframe(adviser_dataframe=adviser_dataframe)
 
         sorted_justifications_df = Adviser.create_adviser_results_dataframe_histogram(
             adviser_type_dataframe=final_dataframe
@@ -82,9 +80,7 @@ class TestAdviser(ReportProcessingTestCase):
             adviser_version=adviser_version, adviser_files=adviser_files
         )
 
-        final_dataframe, info_dataframe, error_dataframe = Adviser.create_summary_dataframes(
-            adviser_dataframe=adviser_dataframe
-        )
+        final_dataframe = Adviser.create_summary_dataframe(adviser_dataframe=adviser_dataframe)
 
         adviser_heatmap_df = Adviser.create_adviser_results_dataframe_heatmap(
             adviser_type_dataframe=final_dataframe, number_days=1

--- a/thoth/report_processing/components/adviser.py
+++ b/thoth/report_processing/components/adviser.py
@@ -53,7 +53,7 @@ class Adviser:
 
     @classmethod
     def aggregate_adviser_results(
-        cls, limit_results: bool = False, max_ids: int = 5, is_local: bool = True, repo_path: Optional[Path] = None,
+        cls, limit_results: bool = False, max_ids: int = 5, is_local: bool = True, repo_path: Optional[Path] = None
     ) -> Dict[str, Any]:
         """Aggregate results stored on Ceph or locally from repo for Thoth components reports.
 
@@ -76,7 +76,7 @@ class Adviser:
             return files
 
         files, counter = cls._aggregate_thoth_results_from_local(
-            repo_path=repo_path, files=files, limit_results=limit_results, max_ids=max_ids,
+            repo_path=repo_path, files=files, limit_results=limit_results, max_ids=max_ids
         )
         _LOGGER.info("Number of files retrieved is: %r" % counter)
 
@@ -122,7 +122,7 @@ class Adviser:
 
     @staticmethod
     def _aggregate_thoth_results_from_ceph(
-        files: Dict[str, Any], store_files: Optional[List[str]] = None, limit_results: bool = False, max_ids: int = 5,
+        files: Dict[str, Any], store_files: Optional[List[str]] = None, limit_results: bool = False, max_ids: int = 5
     ) -> Tuple[Dict[str, Any], int]:
         """Aggregate Thoth results from Ceph."""
         adviser_store = AdvisersResultsStore()
@@ -426,7 +426,7 @@ class Adviser:
 
     @staticmethod
     def store_csv_from_dataframe(
-        csv_from_df: str, ceph_sli: CephStore, file_name: str, ceph_path: str, is_public: bool = False,
+        csv_from_df: str, ceph_sli: CephStore, file_name: str, ceph_path: str, is_public: bool = False
     ) -> None:
         """Store CSV obtained from pd.DataFrame on Ceph.
 
@@ -437,7 +437,7 @@ class Adviser:
         else:
             _LOGGER.info(f"Storing on private bucket... {ceph_path}")
         ceph_sli.store_blob(blob=csv_from_df, object_key=ceph_path)
-        _LOGGER.info(f"Succesfully stored  {df_name} at {ceph_path}")
+        _LOGGER.info(f"Succesfully stored  {file_name} at {ceph_path}")
 
     @staticmethod
     def create_pretty_report_from_json(report: Dict[str, Any], is_justification: bool = False) -> str:

